modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CreationMenu.tsx

```diff
@@ -131,6 +132,7 @@ function CreationMenu({
 						<ClayButton
 							aria-label={primaryItems[0].label}
 							className="nav-btn nav-btn-monospaced"
+							data-qa-id="creationMenuNewButton"
```

Comment:
Could you make sure if this is a duplicate of another data-qa-id? If it is, please double check whether there is possibility of both uses of data-qa-id being on the same page.

Reply:
See above